src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = 40 * days;
+  const discountFirst = 50;
+  const discountSecond = 20;
+  const daysFirst = 7;
+  const daysSecond = 3;
+
+  if (days >= daysFirst) {
+    return sum - discountFirst;
+  } else if (days >= daysSecond) {
+    return sum - discountSecond;
+  }
+
+  return sum;
 }
 
```

Comment:
```suggestion
  const dayCost = 40;
  const sum = dayCost * days;
  const bigDiscount = 50;
  const basicDiscount = 20;
  const daysFirst = 7;
  const daysSecond = 3;

  if (days >= daysFirst) {
    return sum - discountFirst;
  } 
   if (days >= daysSecond) {
    return sum - discountSecond;
  }

  return sum;
}

```

use more semantic names of variables and pass the [`checklist`](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)